src/app/(nextjs_migration)/(authenticated)/user/breaches/page.tsx

```diff
@@ -148,8 +113,7 @@ export default async function UserBreaches() {
       <Script type="module" src="/nextjs_migration/client/js/dialog.js" />
 
       <main data-partial="breaches">
-        {process.env.PREMIUM_ENABLED === "true" &&
-        !isUserSubscribed(subscriptions) ? (
+        {process.env.PREMIUM_ENABLED === "true" && !isUserSubscribed() ? (
```

Comment:
```suggestion
        {process.env.PREMIUM_ENABLED === "true" && !(await isUserSubscribed()) ? (
```

This would've been caught by [this linting rule](https://typescript-eslint.io/rules/no-floating-promises/), so I'll submit a PR to enable it.

Reply:
Yeah thanks I rebased and the linter caught it, nice work :) Already fixed